### PR TITLE
fix(css) add ::cue()  example

### LINKED ISSUE #23209: ::cue should include ::cue() information
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/::cue

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

The `::cue` selector also has a `::cue()` pseudo-element version which accepts a selector. This is not reflected in any examples or explanations on the page.

### What did you expect to see?

an example, such as `::cue([srclang|="en"])` explained

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/_doublecolon_cue`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/::cue
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/_doublecolon_cue/index.md
* Last commit: https://github.com/mdn/content/commit/1a7ab87dc2ce1e8a90b2155eeb13efbc8a50e845
* Document last modified: 2022-09-27T04:25:08.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fixes https://github.com/mdn/content/issues/23209

Merge https://github.com/mdn/interactive-examples/pull/2787 before this PR.

### CODE REVIEW

File: files/en-us/web/css/_doublecolon_cue/index.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ The **`::cue`** [CSS](/en-US/docs/Web/CSS) [pseudo-element](/en-US/docs/Web/CSS/
 
 The properties are applied to the entire set of cues as if they were a single unit. The only exception is that `background` and its longhand properties apply to each cue individually, to avoid creating boxes and obscuring unexpectedly large areas of the media.
 
+The `::cue(u)` selector selects all the [`<u>`](/en-US/docs/Web/HTML/Element/u) elements inside [the cue text](https://raw.githubusercontent.com/mdn/interactive-examples/main/live-examples/media/examples/friday.vtt).
```
Comment (bsmth) [inline files/en-us/web/css/_doublecolon_cue/index.md:16]:
```suggestion
In the example above, the `::cue(u)` selector selects all the [`<u>`](/en-US/docs/Web/HTML/Element/u) elements inside [the cue text](https://raw.githubusercontent.com/mdn/interactive-examples/main/live-examples/media/examples/friday.vtt).
```

File: files/en-us/web/css/_doublecolon_cue/index.md [hunk 2]
```diff
@@ -72,6 +74,25 @@ The following CSS sets the cue style so that the text is white and the backgroun
 }
 ```
 
+### Styling WebVTT internal node objects
+
+Cue text can be decorated using tags `c`, `i`, `b`, `u`, `ruby`, `rt`, `v`, and `lang`. Styling of such tags could be customzed using `::cue()` selector.
```
Comment (bsmth) [inline files/en-us/web/css/_doublecolon_cue/index.md:79]:
```suggestion
Cue text can include _internal node objects_ as the tags (similar to HTML elements) `<c>`, `<i>`, `<b>`, `<u>`, `<ruby>`, `<rt>`, `<v>`, and `<lang>`. The `::cue()` selector can be used to apply styles to these elements to customize how the WebVTT track is displayed.
```

File: files/en-us/web/css/_doublecolon_cue/index.md [hunk 3]
```diff
@@ -72,6 +74,25 @@ The following CSS sets the cue style so that the text is white and the backgroun
...
+Consider following cue text that uses `<u>` tag to underline some text:
```
Comment (bsmth) [inline files/en-us/web/css/_doublecolon_cue/index.md:80]:
```suggestion
Consider the following cue text that uses the `<u>` tag to underline some text:
```

File: files/en-us/web/css/_doublecolon_cue/index.md [hunk 4]
```diff
@@ -72,6 +74,25 @@ The following CSS sets the cue style so that the text is white and the backgroun
...
+
+```plain
+00:00:01.500 --> 00:00:02.999 line:80%
+Tell me, is the <u>lord of the universe</u> in?
+```
+
+Following CSS rule customizes the text decoration inside the `<u>` tag:
```
Comment (bsmth) [inline files/en-us/web/css/_doublecolon_cue/index.md:87]:
```suggestion
The following CSS rule customizes the text inside the `<u>` tag with a color and a [text-decoration](/en-US/docs/Web/CSS/text-decoration):
```

**Review by bsmth — COMMENTED** [review]
Looking good, a couple of comments for you to consider. What do you think?

**Review by bsmth — APPROVED** [review]
Thanks a lot, Onkar! 🙏🏻

**Comment by estelle** [comment]
Thanks Onkar!

**Pull request merged**